l10n_es_sigaus_invoice/models/sigaus_mixin.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from odoo import SUPERUSER_ID, fields, models
+
+
+class SigausMixin(models.AbstractModel):
+    _name = "sigaus.mixin"
+    _description = "Sigaus Mixin"
+
+    is_sigaus = fields.Boolean(compute="_compute_is_sigaus", store=True)
+    sigaus_is_date = fields.Boolean(compute="_compute_sigaus_is_date", store=True)
```

Comment:
Sigaus is date? A que se refiere este campo? Puede que el nombre no sea muy descriptivo

Reply:
Es un campo técnico para saber si la fecha de factura o, en su defecto, la fecha de creación es posterior a la fecha que se ha establecido en la compañía a partir de la que se aplica la aportación SIGAUS.

Reply:
Pon eso (en inglés) en el help